src/databricks/labs/ucx/workspace_access/listing.py

```diff
@@ -40,9 +43,18 @@ def _progress_report(self, _):
                 f" rps: {rps:.3f}/sec"
             )
 
+    def _try_list_workspace(self, path: str) -> Iterator[ObjectType]:
+        try:
+            return self._ws.workspace.list(path=path, recursive=False)
+        except InternalError as e:
+            logger.warning(f"Had an InternalError when listing path {path}, retrying")
```

Comment:
we're already doing debug logging on SDK level: https://github.com/databricks/databricks-sdk-py/blame/main/databricks/sdk/retries.py#L49, no need to double-log here